lmp/build.sh

```diff
@@ -48,6 +48,7 @@ ln -s $cache downloads
 
 chown -R builder .
 
+export FACTORY_SSTATE_CACHE_MIRROR="/var/cache/bitbake/v${LMP_VERSION_CACHE}-sstate-cache"
```

Comment:
Why is this var exported here instead of the same location as previously used by SSTATE_CACHE_MIRROR?

Reply:
Before the `SSTATE_CACHE_MIRROR` is not exported because only used inside `lmp/bb-config.sh`, this new `FACTORY_SSTATE_CACHE_MIRROR` will be used inside `lmp/bb-config.sh` and `lmp/bb-build.sh` so it needs to be exported otherwise will not be seen.
Another advantage of removing it, is that it is no longer possible to modify the mirror using it on `factory-config.yml` and this must be done now modifying the real bitbake variable `SSTATE_MIRRORS`.